tests/simple/force.uplc

```diff
@@ -1 +1 @@
-(program 1.0.0 (force x))
```

Comment:
again, I think we might want to keep tests that involve free variables and create another test with `lam x (force x)`

Reply:
Added `force-lam.uplc` and included `force.uplc` in the failing list.